integration-test/nextjs/src/app/cc/dynamic/useQuery_forcedSuspense/page.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+
+import { useQuery } from "@apollo/experimental-nextjs-app-support/ssr";
+import { enableSSRWaitForUseQuery } from "@apollo/client-react-streaming";
+import type { TypedDocumentNode } from "@apollo/client";
+import { gql, useApolloClient } from "@apollo/client";
+
+const QUERY: TypedDocumentNode<{
+  products: {
+    id: string;
+    title: string;
+  }[];
+}> = gql`
+  query dynamicProducts {
+    products {
+      id
+      title
+    }
+  }
+`;
+
+export const dynamic = "force-dynamic";
+
+export default function Page() {
+  enableSSRWaitForUseQuery(useApolloClient());
```

Comment:
This line should usually be part of `makeClient`, but here I wanted to enable this behaviour just for that one page.